code/__HELPERS/lists.dm

```diff
@@ -640,3 +640,4 @@ proc/dd_sortedObjectList(list/incoming)
 
 // LAZYING PT 2: THE LAZENING
 #define LAZYREINITLIST(L) LAZYCLEARLIST(L); LAZYINITLIST(L);
+#define LAZYQDELLIST(L) if(L) { for(var/o in L) qdel(o); L.Cut() }

```

Comment:
Oh wait, this is the entire schtick of all those `LAZY` macros - they don't care if the list is empty or not